security/certificate_types_descriptions/proxy-certificates.adoc

```diff
@@ -64,9 +64,7 @@ The user-provided trust bundle is represented as a config map. The config map is
 
 Complete proxy support means connecting to the specified proxy and trusting any signatures it has generated. Therefore, it is necessary to let the user specify a trusted root, such that any certificate chain connected to that trusted root is also trusted.
 
-If using the RHCOS trust bundle, place CA certificates in `/etc/pki/ca-trust/source/anchors`.
-
-See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-shared-system-certificates_security-hardening[Using shared system certificates] in the Red Hat Enterprise Linux documentation for more information.
+If you use the {op-system} trust bundle, place CA certificates in `/etc/pki/ca-trust/source/anchors`. See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/securing_networks/using-shared-system-certificates_securing-networks[Using shared system certificates] in the _Securing networks_ document ({op-system-base-full} documentation).
```

Comment:
this resolves as "...Securing networks document (Red Hat Enterprise Linux (RHEL) documentation)."

Which has a double parentheses. I would try to avoid that if possible.

We also do often prefer the "For more information, see ..." wording, if you want to consider that. 

Adding potential suggestion below, but I'm sure there are other ways could be done.


```suggestion
If you use the {op-system} trust bundle, place CA certificates in `/etc/pki/ca-trust/source/anchors`. For more information, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/securing_networks/using-shared-system-certificates_securing-networks[Using shared system certificates] in the {op-system-base-full} _Securing networks_ document.
```